common/src/main/java/com/wynntils/utils/render/MapRenderer.java

```diff
@@ -33,10 +33,21 @@
 
 public final class MapRenderer {
     // Zoom is the scaling of the map. The bigger the zoom, the more detailed the map becomes.
+
+    // ZOOM_STEPS is the number of steps the zoom can take. (0 to ZOOM_STEPS - 1)
```

Comment:
I'm not sure I'm following this. I see you introduced the term "zoom steps", I assume this is to separate it from the value that you multiply with when drawing the map. This is ... good, I guess. 

But I don't understand how 40, 0.1 and 5 relates. Or what MIN_ZOOM_LOG etc is supposed to do.

I would like to stipulate that we have a min zoom level of 1, and a max zoom level of 100. (Or the other way around, I'm not sure which is most natural.) Then the other values should be calculated based from this, to see how we should multiply the map, what zoom level to start at, etc.

Reply:
Also, I don't think we should represent zoom as an int. Even if we go from 1 to 100, we should be able to have a zoom of e.g. 2.5. How much the zoom value changes when you scroll the mouse well is kind of independent from everything else. (In fact, maybe it should be adjustable depending on how "fast" zoom you want?)

In a perfect world, this might also make us support a touchpad-style pinch-to-zoom. That'd be cool!

Reply:
I've just pushed the second iteration of this code.

I've made zoom steps represent map zoom as a percentage between 1-100 (0% zoom makes no sense).
I've also fine-tuned the default values for scrolling so that it feels like to old map.
I've also added float-steps, making this world perfect, as we happen to support touchpad-style pinch-to-zoom by concept. :)